src/main/java/org/opensearch/knn/training/TrainingJob.java

```diff
@@ -90,7 +91,8 @@ public TrainingJob(
                 "",
                 nodeAssignment,
                 knnMethodContext.getMethodComponentContext(),
-                knnMethodConfigContext.getVectorDataType()
+                knnMethodConfigContext.getVectorDataType(),
+                Version.CURRENT
```

Comment:
Can we set version during knnMethodConfigContext configuration?

Reply:
Oh I didn't even see that, we should just be able to call getVersionCreated()